purchase_reception_status/__manifest__.py

```diff
@@ -11,7 +11,8 @@
     "author": "Akretion,Odoo Community Association (OCA)",
     "maintainers": ["alexis-via"],
     "website": "https://github.com/OCA/purchase-workflow",
-    "depends": ["purchase"],
+    "depends": ["purchase_stock"],
+    "external_dependencies": {"python": ["openupgradelib"]},
```

Comment:
@bizzappdev 
It's all good, but please don't add a dependency to the external library "openupgradelib".
Our customers don't have that library installed and we don't want to add it just for this small SQL update.
You can just use `env.cr.execute()` in the migrate.